cli/cmd/plugin/diagnostics/pkg/collect_bootstrap.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020-2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
```

Comment:
Nit: these could just have `2021` instead of `2020-2021`. But I don't think there's really any issue if we go back that far.